src/main/resources/hudson/plugins/report/genericchart/ChartModel/help-resultWhiteList.html

```diff
@@ -0,0 +1,4 @@
+<div>
+	Packages whitelisted (java regex, space separated) are shown in results (to keep zoomed reuslts readable) and/or fitler specific brach. For example:<br/>
+	<span style="font-family: monospace">.*2\.6\.6\.0.*</span><br/>
+</div>
```

Comment:
This is just copypasted.
Packages whitelisted (java regex, space separated) are shown in results (to allow filtering of individual job IDs) and/or show only specific brach or simialrly. For example:<br/>
+	<span style="font-family: monospace">.*2\.6\.6\.0.*</span><br/>